build.gradle

```diff
@@ -117,6 +118,21 @@ task codeCoverageReport(type: JacocoReport) {
     }
 }
 
+task createProperties(dependsOn: processResources) {
+    doLast {
+        new File("$project.rootDir/persist-cli/src/main/resources/version.properties").withWriter { w ->
+            Properties properties = new Properties()
+            properties['version'] = '${project.version}'
+            properties['persistVersion'] = persistNativeVersion.toString()
+            properties.store w, null
+        }
+    }
+}
```

Comment:
Cant we change the logic to update the file instead of creating it every build? This updates the created timestamped even if the values doenst change,

Reply:
Updated in https://github.com/ballerina-platform/persist-tools/pull/154